chapters/Gosses1.md

```diff
@@ -0,0 +1,39 @@
+# Standing on the Shoulders of the Guy in the UK Office
+ 
+Justin Gosses
+
+> Writing in progress....  &mdash; 
+
+
+The phrase “If I have seen further, it is by standing on the shoulders of giants.” was written by Isaac Newton to Robert Hooke in reference to building on the work his predecessors. In science, we often think of these giants as long dead men in history books. In computer science, many of the giants are still alive.  In open-source, they are middle aged to young people you end up drinking beer with at conferences. In intra-source, they are your co-workers writing code right now, including those you’ve never met. 
```

Comment:
Re `giant shoulders`, I find it super interesting that Newton might have been throwing down an insult. From wiki:

> This has recently been interpreted by a few writers as a sarcastic remark directed at Hooke's appearance.[8] Although Hooke was not of particularly short stature, he was of slight build and had been afflicted from his youth with a severe kyphosis. However, at this time Hooke and Newton were on good terms and had exchanged many letters in tones of mutual regard. Only later, when Robert Hooke criticized some of Newton's ideas regarding optics, was Newton so offended that he withdrew from public debate. The two men remained enemies until Hooke's death.